### ISSUE TITLE
Favicon.ico missing

### ISSUE THREAD

**Issue by rugk** [yourfirstpr/yourfirstpr.github.io#147]
In the network console, I can see Firefox is just always getting a 404 error when trying to request a favicon in `favicon.ico`. :smiley:

**Comment by A1337CBS** [comment]
I opened the page yourfirstpr.github.io in Firefox and could not see this issue so  can you be more specific as to where are you seeing this issue?

**Comment by rugk** [comment]
Yeah, it's only one request, but getting away from these technical details: The issue here is, that the website has no favicon. No website™ should have no favicon.

**Comment by gonaicsoft** [comment]
Has it been resolved or can I work on it?

**Comment by rugk** [comment]
If I visit https://yourfirstpr.github.io/ I still see no favicon, so AFAIK/as it seems, it does not seem to be resolved.

**Comment by shivamkainth** [comment]
I am going to work on this.

**Comment by doghouch** [comment]
> If I visit https://yourfirstpr.github.io/ I still see no favicon, so AFAIK/as it seems, it does not seem to be resolved.

The only thing that is required is to put a favicon file ("favicon.ico") in the root directory of the GitHub page.

:)

**Comment by RamanaVem** [comment]
Is this still open?

**Comment by rugk** [comment]
there are already two PRs that (at least claim to :wink:) solve this… (linked above)
